packages/vsce/src/Utilities.ts

```diff
@@ -71,7 +74,8 @@ export function registerCommands(context: vscode.ExtensionContext): vscode.Dispo
         }),
         vscode.commands.registerCommand("zowe-native-proto-vsce.restart", async (profName?: string) => {
             imperative.Logger.getAppLogger().trace("Running restart command for profile %s", profName);
-            const profile = await SshConfigUtils.promptForProfile(profName);
+            const vscePromptApi = new VscePromptApi(await profCache.getProfileInfo());
```

Comment:
Just out of curiosity, what happens if we decide to reuse the ProfileInfo instance here with Line 61 (or 98)? 😋

I'm not suggesting to try it, but in theory could reduce the number of times we are loading profileInfo 😋

Reply:
oh, these are `registerCommand` function calls 😅 
that makes sense 😋 
we would want to get the most up-to-date profiles from disk 😋